airbyte-db/db-lib/src/main/java/io/airbyte/db/jdbc/JdbcUtils.java

```diff
@@ -107,6 +107,9 @@ public static Map<String, String> parseJdbcParameters(final String jdbcPropertie
         final String[] split = kv.split("=");
         if (split.length == 2) {
           parameters.put(split[0], split[1]);
+        }
+        else if (split.length == 3 && kv.contains("sessionVariables") {
```

Comment:
@marcosmarxm Sure, let me know how I should adapt the format. This seems to be an outdated commit, I fixed a syntax error in https://github.com/airbytehq/airbyte/pull/25386/commits/3985a6c2a3725c5a0afb8ad5cbeaa812dea12f80. Could you give me a hand with the unit tests? I think testing that the string `sessionVariables=max_execution_time=10000` would be a good regression test. Otherwise, I'll see what I can do.